### PR TITLE
Fix pgx_pgmq read

### PR DESCRIPTION
**Pull request by ianstanton**
Read function parameters recently changed. Updating in `pgx_pgmq` extension.

### CODE REVIEW

**Comment by ianstanton** [comment]
Need to investigate failures after merging in main (CI was green)

**Comment by ianstanton** [comment]
`pgmq` dependency was reverted to `0.4.0`

**Review by sjmiller609 — APPROVED** [review]
I would increment the version of pgx_pgmq to 0.0.4 or 0.5.1

File: extensions/pgx_pgmq/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pgx_pgmq"
-version = "0.0.3-alpha.1"
+version = "0.5.1"
```
Comment (ChuckHend) [inline extensions/pgx_pgmq/Cargo.toml:3]: Are we going to tie these versions together? I'd expect the crates development to slow over time and the extension to pick up.

Comment (ianstanton) [reply extensions/pgx_pgmq/Cargo.toml:3]: Let's go with `0.0.4` in that case

**Review by ChuckHend — APPROVED** [review]

**Pull request merged**